modules/apps/feature-flag/feature-flag-web/src/main/resources/META-INF/resources/js/FeatureFlagToggle.tsx

```diff
@@ -8,30 +8,32 @@ import React, {useState} from 'react';
 
 interface IProps {
 	ariaDescribedBy: string;
-	companyId: number;
+	companyId?: number;
```

Comment:
`CompanyId` is always present, it is not optional.

Reply:
Fixed